### PR TITLE
Feature/stronger headlines

### PR DESCRIPTION
**Pull request by ayurmedia**
Add CSS to Highlight H1 , H2 and make it easier to read in Ticket-Body Text.

### CODE REVIEW

File: Stronger Headlines/StrongerHeadlines.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+tau.mashups
+    .addDependency('tp3/mashups/topmenu')
```
Comment (tsarevichdm) [inline Stronger Headlines/StrongerHeadlines.js]: Looks like no dependencies needed. Please, remove them.

Comment (ayurmedia) [reply Stronger Headlines/StrongerHeadlines.js]: done

Comment (tsarevichdm) [reply Stronger Headlines/StrongerHeadlines.js]: and `.addDependency('tau/core/bus.reg')` ?

Comment (ayurmedia) [reply Stronger Headlines/StrongerHeadlines.js]: not needed, this mashup only applies global css rules.

Comment (ayurmedia) [reply Stronger Headlines/StrongerHeadlines.js]: removed

Comment (tsarevichdm) [reply Stronger Headlines/StrongerHeadlines.js]: Thank you. Merging...

**Pull request merged**